__tests__/app.test.js

```diff
@@ -53,14 +55,43 @@ describe("Set of GET Tests", () => {
           expect(response.body).toHaveProperty("GET /api/articles");
         });
     });
-    test("should include description key for each endpoint", () => {
+    test("should match received endpoints with endpoints.json", () => {
+      const endpointsPath = "./endpoints.json";
+      const expectedEndpoints = JSON.parse(
+        fs.readFileSync(endpointsPath, "utf-8")
+      );
       return supertest(app)
         .get("/api/endpoints")
         .then((response) => {
-          const endpoints = response.body;
-          Object.keys(endpoints).forEach((endpoint) => {
-            expect(endpoints[endpoint]).toHaveProperty("description");
-          });
+          expect(response.body).toEqual(expectedEndpoints);
+        });
+    });
+  });
+  describe("GET Articles by ID", () => {
+    test("should return a Status Code: 200 for a successful request", () => {
+      return supertest(app).get("/api/articles/5").expect(200);
+    });
+    test("should return a Status Code: 404 for a non-existent article ID", () => {
+      return supertest(app).get("/api/articles/888").expect(404);
+    });    
+    test("should return a Status Code: 400 if passed article ID is not a number", () => {
+      return supertest(app).get("/api/articles/semih8").expect(400);
+    });
+    test("should return an article with the Correct Keys In", () => {
+      return supertest(app)
+        .get("/api/articles/8")
+        .expect(200)
+        .then((response) => {
+          const article = response.body.article[0];
```

Comment:
your response should be an object containing a key of article with the article object, not an array containing an object :)